package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build",
+    "build:lib": "ng build ngx-flow",
```

Comment:
🙏 Please update also this part of docs: https://github.com/flowjs/ngx-flow#development

Reply:
I updated